schema/fields.go

```diff
@@ -413,10 +417,66 @@ func FindIndex(indexes []*Index, name string) *Index {
 	return nil
 }
 
+type FieldMultiType struct {
+	Type       []string
+	isNullable bool
+}
+
+func (f *FieldMultiType) append(val string) {
+	if val == "null" {
+		f.isNullable = true
+	} else {
+		f.Type = append(f.Type, val)
```

Comment:
Type can only be `1` as you have `isNullable` property to support "null". Then do we need to have `Type` as an array, or can it be just a string?